### PR TITLE
Devel ergo cub sn001

### PR DESCRIPTION
**Pull request by pattacini**
Update upstream ergoCubSN001 conf files.

### CODE REVIEW

File: ergoCubSN001/calibrators/left_arm-calib.xml
Comment (pattacini) [inline ergoCubSN001/calibrators/left_arm-calib.xml:34]: Hi @S-Dafarra @GiulioRomualdi 
Any specific reason why joint 9 is not included?

File: ergoCubSN001/ergocub.xml
Comment (pattacini) [inline ergoCubSN001/ergocub.xml:75]: Do we want to remove these lines?

**Review by pattacini — COMMENTED** [review]
Just a few inline comments before merging.

File: ergoCubSN001/calibrators/left_arm-calib.xml
Comment (S-Dafarra) [reply ergoCubSN001/calibrators/left_arm-calib.xml:34]: That is the index abduction. It has been removed by @AntonioConsilvio in https://github.com/robotology/robots-configuration/pull/681/commits/cff4ac3e282bdaea6e94c14ed3ac8028a840837c

@AntonioConsilvio I suppose it is still broken right?

File: ergoCubSN001/ergocub.xml
Comment (S-Dafarra) [reply ergoCubSN001/ergocub.xml:75]: I prefer not. That is a just a temporary hack since both arm FTs are broken at the moment

File: ergoCubSN001/calibrators/left_arm-calib.xml
Comment (pattacini) [reply ergoCubSN001/calibrators/left_arm-calib.xml:34]: Gotcha!

File: ergoCubSN001/ergocub.xml
Comment (pattacini) [reply ergoCubSN001/ergocub.xml:75]: Gotcha!

**Comment by pattacini** [comment]
> Let me know if you want to proceed anyhow. If you do, can we avoid to squash? In this way it gets easy to revert them when the robot is fixed

We need to avoid squashing anyhow for the reason reported in par. 5.i at https://github.com/robotology/robots-configuration/blob/master/.github/CONTRIBUTING.md#-fast-pace-local-workflow.

I may try to cherry-pick the other commits in upstream@devel and check whether this PR will get updated with the remaining commits. Once verified, I can close the PR unmerged.

Does it sound ok?

**Comment by pattacini** [comment]
>I may try to cherry-pick the other commits in upstream@devel and check whether this PR will get updated with the remaining commits. Once verified, I can close the PR unmerged.

Or even better, I could merge the PR and then revert those two commits.

**Comment by S-Dafarra** [comment]
I have booked the robot at 13. I can rebase the branch on upstream@devel, and back up those two commits in another branch. Then, we can merge this (without the two commits) and I will take care of bringing those commits back into ``devel-ergoCubSN001``. Would that work?

**Comment by pattacini** [comment]
Perfect! Go ahead this way.

**Review by S-Dafarra — APPROVED** [review]
Done @pattacini, the PR is ready to go

**Comment by pattacini** [comment]
Thanks!
I'll merge it after the CI.

**Comment by S-Dafarra** [comment]
One thing that I noticed is that the bot has push-force permission. I think it might be better to protect the branch from push force and allow only a handful of people doing it

**Comment by S-Dafarra** [comment]
Awesome! I will test it immediately

**Comment by S-Dafarra** [comment]
I rebased on top of devel and I force pushed. It wrote the following
```
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
remote: Bypassed rule violations for refs/heads/devel-ergoCubSN001:
remote:
remote: - Cannot force-push to this branch
remote:
To https://github.com/icub-tech-iit/robots-configuration.git
 + a51affcbb...5081ba1fa devel-ergoCubSN001 -> devel-ergoCubSN001 (forced update)
```

In this case they should have been aligned, so it worked anyhow I suppose

**Comment by S-Dafarra** [comment]
I tried to remove some commit and force pushed and it worked again, same message. Also in this case there were 0 deltas anyhow.

**Comment by pattacini** [comment]
The message `cannot force-push` tells us that you cannot force-push when being identified as the bot doing that.
If you try out the same being yourself, we shouldn't get any failure.

**Comment by S-Dafarra** [comment]
> The message `cannot force-push` tells us that you cannot force-push when being identified as the bot doing that. If you try out the same being yourself, we shouldn't get any failure.

I did not enter my credentials, I suppose I pushed as bot

**Pull request merged**
